store/tikv/kv.go

```diff
@@ -185,15 +185,15 @@ func (s *tikvStore) SendKVReq(req *pb.Request, region *requestRegion) (*pb.Respo
 		req.Context = region.GetContext()
 		resp, err := client.SendKVReq(req)
 		if err != nil {
-			log.Warnf("send tikv request error: %v, try next store later", err)
-			s.regionCache.NextStore(region.GetID())
+			log.Warnf("send tikv request error: %v, try next peer later", err)
+			s.regionCache.NextPeer(region.GetID())
 			continue
 		}
 		if regionErr := resp.GetRegionError(); regionErr != nil {
 			// Retry if error is `NotLeader`.
 			if notLeader := regionErr.GetNotLeader(); notLeader != nil {
 				log.Warnf("tikv reports `NotLeader`: %s, retry later", notLeader.String())
-				s.regionCache.UpdateLeader(notLeader.GetRegionId(), notLeader.GetLeaderStoreId())
+				s.regionCache.UpdateLeader(notLeader.GetRegionId(), notLeader.GetLeader().GetId())
```

Comment:
@siddontang No matter. Protobuf checks if m==nil, so it will return 0 if GetLeader returns nil.